setup.py

```diff
@@ -67,30 +63,6 @@ def read_reqs(fname):
     "gcs": gcs_reqs,
 }
 
-# Tox has a weird issue where it can't import pip from it's virtualenv when skipping normal installs
-if not bool(int(os.environ.get('SKIP_PIP_CHECK', 0))):
-    pip_too_old = False
-    pip_message = ''
-    try:
-        import pip
-
-        pip_version = tuple([int(x) for x in pip.__version__.split('.')[:3]])
-        pip_too_old = pip_version < (9, 0, 1)
-        if pip_too_old:
-            # pip is too old to handle IPython deps gracefully
-            pip_message = (
-                'Your pip version is out of date. Papermill requires pip >= 9.0.1. \n'
-                'pip {} detected. Please install pip >= 9.0.1.'.format(pip.__version__)
-            )
-    except Exception:
-        # We only want to optimistically report old versions
-        pass
-
-    if pip_message:
-        print(pip_message, file=sys.stderr)
-        sys.exit(1)
-
-
```

Comment:
This was for an old version of pip and previously needing IPython on start, so it makes sense to remove this. 👍